Bio/PDB/MMCIFParser.py

```diff
@@ -141,7 +143,12 @@ def _build_structure(self, structure_id):
             altloc = alt_list[i]
             if altloc in _unassigned:
                 altloc = " "
-            int_resseq = int(seq_id_list[i])
+            try:
+                int_resseq = int(seq_id_list[i])
+            except ValueError:
+                raise PDBConstructionException(
+                    "try parser.get_structure('structure_id', 'file_name', use_label_seq_id=True)"
```

Comment:
Perhaps clarify this error, something like "conversion of sequence ID to integer failed, consider using...".